build/pkgs/pycryptosat/SPKG.rst

```diff
@@ -4,8 +4,21 @@ pycryptosat: Python module of cryptominisat
 Description
 -----------
 
-   Build and install pycryptosat into appropriate venv.
-   See cryptominisat for more details.
+   Build and install pycryptosat (Python frontend to CryptoMiniSat, which will be built if needed too)
+   into appropriate venv.
+
+
+
+   CryptoMiniSat is a SAT solver that aims to become a premiere SAT solver
+   with all the features and speed of successful SAT solvers, such as MiniSat
+   and PrecoSat. The long-term goals of CryptoMiniSat are to be an efficient
+   sequential, parallel and distributed solver. There are solvers that are
+   good at one or the other, e.g. ManySat (parallel) or PSolver (distributed),
+   but we wish to excel at all.
+
+   CryptoMiniSat 2.5 won the SAT Race 2010 among 20 solvers submitted by researchers and industry.
+
```

Comment:
Please unindent for proper formatting in the HTML documentation.

Also, the description should just describe the package, not talk about venvs etc.

Reply:
I haven't added anything about venvs, it was there.

Please edit and add a commit as you see fit.

Reply:
fixed by [6d3d782](https://github.com/sagemath/sage/pull/37669/commits/6d3d782ccf21a07496c35d4b415195fb43d433e2)